src/systemctl/systemctl-is-active.c

```diff
@@ -57,6 +57,7 @@ int verb_is_active(int argc, char *argv[], void *userdata) {
         static const UnitActiveState states[] = {
                 UNIT_ACTIVE,
                 UNIT_RELOADING,
+                UNIT_REFRESHING,
```

Comment:
btw, the output_units_list() function in systemctl-units-list.c also needs updating for this new high-level state, for coloring output. I guess wherever we compare the state with "reloading" we should treat "refreshing" the same when it comes to coloring.

some cases in src/systemctl/systemctl-show.c too